pkg/optimizely/interface.go

```diff
@@ -17,7 +17,18 @@
 // Package optimizely wraps the Optimizely SDK
 package optimizely
 
+import (
+	optimizelyconfig "github.com/optimizely/go-sdk/pkg/config"
+)
+
 // Cache defines a basic interface for retrieving an instance of the OptlyClient keyed off of the SDK Key
 type Cache interface {
 	GetClient(sdkKey string) (*OptlyClient, error)
 }
+
+// SyncedConfigManager has the basic ConfigManager methods plus the SyncConfig method to trigger immediate updates
+type SyncedConfigManager interface {
```

Comment:
is this SyncedConfigManager created only to help with testing ?

Reply:
Yes. The `SyncConfig` method only exists on the `PollingConfigManager` so we've been using that explicit reference, this abstracts that out a bit for testing and defines only the methods we're actually using.